coil-base/src/main/java/coil/request/DefaultRequestOptions.kt

```diff
@@ -26,6 +26,7 @@ class DefaultRequestOptions(
     val bitmapConfig: Bitmap.Config = Utils.DEFAULT_BITMAP_CONFIG,
     val allowHardware: Boolean = true,
     val allowRgb565: Boolean = false,
+    val premultipliedAlpha: Boolean = true,
```

Comment:
I would only add this attribute to `ImageRequest` and `Options` unless you feel there's a good use case for setting the `ImageLoader`'s default. For context, we don't support setting the default `ColorSpace` for an `ImageLoader` either.

Reply:
👍